src/index.tsx

```diff
@@ -3,10 +3,13 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import {BrowserRouter} from 'react-router-dom'
```

Comment:
`;` missing at the end.

anyway, that's until we get lint working, such fixes should be automatically applied